test/powershell/Host/TabCompletion/TabCompletion.Tests.ps1

```diff
@@ -1345,7 +1345,7 @@ Describe "WSMan Config Provider tab complete tests" -Tags Feature,RequireAdminOn
     ) {
         param($path, $parameter, $expected)
         $script = "new-item wsman:\$path $parameter"
-        $res = TabExpansion2 -inputScript $script -cursorColumn $script.Length
+        $res = TabExpansion2 -inputScript $script
```

Comment:
Do we still have a test where the parameter is used? Ideally the number of cases we test monotonically increases